### PR TITLE
Correcting typo (TransportLater actually sounds nice!)

### PR DESCRIPTION
**Pull request by Lompandi**

### CODE REVIEW

**Comment by Adrian8115** [comment]
Don't forget to run rustfmt

**Comment by Adrian8115** [comment]
Also, could you please remove the pure udp transport stuff, it is not planned anymore and unneeded

File: crates/proto/src/error.rs [hunk 1]
```diff
@@ -71,6 +71,8 @@ pub enum TransportLayerError {
     IOError(#[from] Arc<IOError>),
     #[error("Raknet UDP Error: {0}")]
     RaknetUDPError(#[from] RaknetError),
+    #[error("UDP Error: {0}")]
+    UDPError(#[from] UdpError),
```
Comment (theaddonn) [inline crates/proto/src/error.rs:75]: remove udp stuff please

File: crates/proto/src/error.rs [hunk 2]
```diff
@@ -84,3 +86,11 @@ pub enum RaknetError {
     #[error("Format Error: {0}")]
     FormatError(String),
 }
+
+#[derive(Error, Debug, Clone)]
+pub enum UdpError {
```
Comment (theaddonn) [inline crates/proto/src/error.rs:91]: remove udp stuff please

File: crates/proto/src/transport_layer/connection.rs [hunk 3]
```diff
@@ -12,8 +12,8 @@ pub enum TransportLayerConnection {
     // TODO RaknetTCP(...),
     NetherNet(/* TODO */),
     // TODO Quic(s2n_quic::connection::Connection),
-    // TODO Tcp(net::TcpStream),
-    // TODO Udp(net::UdpSocket)
+    // Tcp(std::net::TcpStream),
+    Udp(tokio::net::UdpSocket) //MCBE over UDP :sobb:??
```
Comment (theaddonn) [inline crates/proto/src/transport_layer/connection.rs:16]: remove udp stuff please

File: crates/proto/src/transport_layer/connection.rs [hunk 4]
```diff
@@ -34,6 +34,22 @@ impl TransportLayerConnection {
                     .await
                     .map_err(|e| TransportLayerError::RaknetUDPError(RaknetError::SendError(e)))
             }
+            TransportLayerConnection::Udp(conn) => {
```
Comment (theaddonn) [inline crates/proto/src/transport_layer/connection.rs:37]: remove udp stuff please

File: crates/proto/src/transport_layer/connection.rs [hunk 5]
```diff
@@ -81,6 +130,9 @@ impl TransportLayerConnection {
             TransportLayerConnection::RaknetUDP(conn) => {
                 conn.close().await;
             }
+            TransportLayerConnection::Udp(socket) => {
```
Comment (theaddonn) [inline crates/proto/src/transport_layer/connection.rs:133]: remove udp stuff please

**Review by Adrian8115 — CHANGES_REQUESTED** [review]
remove udp stuff please

**Pull request closed without merging**